package.json

```diff
@@ -123,9 +125,9 @@
         "types": "./esm/identity/index.d.ts",
         "module": "./esm/identity/index.js",
         "import": "./esm/identity/index.js",
-        "default": "./lib/identity/index.js"
+        "default": "./esm/identity/index.js"
```

Comment:
I was able to make onchainkit work with vocs with this change.

Reply:
cc @wespickett

Reply:
I think that's a massive breaking changes. And we should avoid on doing that.

Reply:
Let's consider this breaking change only at v1 cut discussion, which will be in late May.